docs/ad/api.md

```diff
@@ -162,24 +173,69 @@ POST _opendistro/_anomaly_detection/detectors/<detector_id>/_preview
 {
   "anomaly_result": [
     {
-      "detector_id": "B2aJlG4B04EFYPipCRO4",
-      "anomaly_grade": 0.1,
-      "confidence": 0.7,
+      "detector_id": "m4ccEnIBTXsGi3mvMt9p",
+      "data_start_time": 1588843020000,
+      "data_end_time": 1588843620000,
       "feature_data": [
         {
-          "feature_id": "value_sum",
-          "feature_name": "Value Sum",
-          "data": 20
+          "feature_id": "xxokEnIBcpeWMD987A1X",
+          "feature_name": "total_order",
+          "data": 489.9929131106
         }
       ],
-      "data_start_time": 1573514220000,
-      "data_end_time": 1573514520000
+      "anomaly_grade": 0,
+      "confidence": 0.99
     }
   ],
```

Comment:
anomaly_result has multiple items, should we add `...` before `]` ?

Reply:
Added.